### DISCUSSION TITLE
The rtsp-simple -server process automatically exits at intervals（ARM v8）

### DISCUSSION THREAD

**Discussion by DreamyCheng** [bluenviron/mediamtx#1016] [Questions]
The rtsp-simple -server process automatically exits at intervals（ARM v8）

**Comment by markbui123** [comment]
I'm new, and I seem to have the same problem.
I use centos 8.
Follow the instructions to run with the command line:
`docker run --rm -it -e RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 -p 8888:8888 aler9/rtsp-simple-server`

**Comment by aler9** [comment]
Please open an issue and provide instructions on how to replicate the problem.
